src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();

```

Comment:
Static methods are generally considered bad practice in object-oriented programming as they can lead to issues with inheritance and overriding. It's better to create an instance of the class where the method you want to call is defined.